### PR TITLE
Update role-ids.md

### PR DESCRIPTION
**Pull request by andrewjhaz**
I made my first change to this repository which I thought would help people understand what was in the role code file.

### CODE REVIEW

**Comment by andrewjhaz** [comment]
Wondered if I should at-mention @aharriso11 in this to see what happens.

File: nhsd-ord-api/role-ids.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Role IDs
 
+This is a list of the role IDs returned by the NHS ORD API.
```
Comment (aharriso11) [inline nhsd-ord-api/role-ids.md]: I think we could also add a line about how people can get this data for themselves.

Comment (andrewjhaz) [reply nhsd-ord-api/role-ids.md]: > I think we could also add a line about how people can get this data for themselves.

No problem, I have done this if you would be able to review?

**Review by aharriso11 — APPROVED** [review]
@andrewjhaz Thanks, I like this and will accept into the main branch.

**Comment by aharriso11** [comment]
Merged.

**Pull request merged**